marketplace/manifest.yaml

```diff
@@ -1,11 +1,11 @@
-applicationName: com.spinkube.spinkube
-publisher: "Spinkube"
+applicationName: dev.spinkube.spinkube
+publisher: "Fermyon"
```

Comment:
As I understand it, the publisher/marketplace partner needs to be a legal entity, so have updated it to Fermyon.  However, if this understanding is incorrect or we can indeed use the SpinKube org, we can change this back.

Reply:
yup sounds good!